doc/monitoring/when_use_cartridge.rst

```diff
@@ -0,0 +1,221 @@
+.. _when_use_cartridge:
+
+When use Cartridge
+==================
+
+.. _when_use_cartridge-getting_started:
+
+Getting started
+---------------
+
+Если вы используете фреймворк `Cartridge <https://www.tarantool.io/en/doc/latest/book/cartridge/>`__
+для разработки своего приложения, в него уже добавлена перманентная роль metrics, которая подключается по
+умолчанию. Всё, что нужно — указать её в cartridge.cfg и сконфигурировать. Это делается так:
+
+1. Добавляем 'cartridge.roles.metrics' в список ролей cartridge.cfg:
+
+.. code-block:: lua
+
+    local ok, err = cartridge.cfg({
+        roles = {
+            ...
+            'cartridge.roles.metrics',
+            ...
+        },
+    })
+
+2. Добавляем секцию в конфигурацию кластера (либо можно использовать
+`set_export <https://www.tarantool.io/en/doc/latest/book/monitoring/api_reference/#lua-function.metrics.set_export>`__).
```

Comment:
I think we may move this below the example

Reply:
Moved it down and added it to the hint block so you can see it better